### PR TITLE
Lifecycle Event Spec documentation

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Added documentation of the Application Lifecycle Event spec.

## Type of documentation

- [x] New documentation
- [ ] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Application Lifecycle Events documentation](https://www.notion.so/rudderstacks/Application-Lifecycle-Event-Spec-96fe23968561404eb4ccea328c6b9a86)

### CODE REVIEW

**Review by desusai7 — APPROVED** [review]
LGTM

File: rudderstack-api/api-specification/application-lifecycle-events-spec.mdx [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+# slug: "/docs/rudderstack-api/api-specification/video-specification"
+title: "Application Lifecycle Events Specification"
+description: >-
+  Detailed technical description of the various application lifecycle events tracked by the RudderStack SDKs.
+---
+
+# Application Lifecycle Events Specification
+
+RudderStack lets you track and capture various application lifecycle events across the mobile SDKs and get insights into various app-related metrics like installs, opens, etc. This guide provides the details and semantic definitions of these events and the associated properties.
```
Comment (pallabmaiti) [inline rudderstack-api/api-specification/application-lifecycle-events-spec.mdx:10]: I think it should change to `capture and track`.

File: rudderstack-api/api-specification/application-lifecycle-events-spec.mdx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+## Supported lifecycle events
+
+RudderStack supports and **automatically tracks** the following application lifecycle events:
```
Comment (pallabmaiti) [inline rudderstack-api/api-specification/application-lifecycle-events-spec.mdx:14]: I think we can remove `supports` work.

File: rudderstack-api/api-specification/application-lifecycle-events-spec.mdx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+RudderStack tracks the following properties for this event:
+
+| Property | Type | Description |
+| :--------| :-----| :------------|
+| `version` | String | The version of the installed application. |
+
+## Application Opened
+
+This event is fired when the user launches the application after the first open and subsequently when the app is reopened every time after it is closed.
```
Comment (pallabmaiti) [inline rudderstack-api/api-specification/application-lifecycle-events-spec.mdx:56]: closed and backgrounded.

**Review by pallabmaiti — APPROVED** [review]

**Pull request merged**